### ISSUE TITLE
select all under Filters not working

### ISSUE THREAD

**Issue by mberend** [downthemall/downthemall#17]
### Your DownThemAll! and Browser Version and Operating System
v3.0.2  ff 47.0.1 Windows 7

### Expected Behavior
under Filters > select all

### Actual Behavior
It doesn't select links

### Additional info such as log extracts
*Please enable Diagnostic Logging in the Privacy tab of the DownThemAll! preferences and either copy/paste essential part of the log or attach the entire log. Please note that the log may include sentitive information which you have to remove prior to posting it!*

**Comment by nmaier** [comment]
So the "All Files" item is still listed under filters, just does not select all links?

**Comment by mberend** [comment]
yes. it's present, but nothing is selected. I'll have to do it manually.

**Comment by nmaier** [comment]
Opening the Preferences/Filters tab and then the All Files filters has what Extensions value?

**Comment by nmaier** [comment]
Also, does this happen on any website or just specific ones?

**Comment by nmaier** [comment]
Oh, or are you actually referring to the "Select All" menu item?

**Comment by mberend** [comment]
the value was EMPTY so I put back the standard value and it's working again!

The standard value is /.*/i

**Comment by mberend** [comment]
thx

**Comment by nmaier** [comment]
I wonder tho WHY the value was empty...

**Comment by mberend** [comment]
Before the update it worked. It happend after the update.

I can't remember if I changed something in that value. I'm using Downthemall for a couple of years now.

**Comment by grmblgrmbl** [comment]
Same here. When selecting any of the filters, nothing in the link list is actually selected.

It's even weirder:

1. Select a link manually.
2. de-select it again
3. Check or uncheck any of the filters

Result: The link selected in step 1 is selected again, even when it does not match the filter.

**Comment by nmaier** [comment]
@grmblgrmbl is the Extensions value in Preferences/Filters empty too for you?

**Comment by grmblgrmbl** [comment]
I just checked, and only some filters have extensions. Looks like the update has lost a few settings.

If I add eg. PDFs to "´Documents", then PDF as selected when using this filter. So it seems it's a matter of missing extensions.

The weird behaviour, however, remains.

**Comment by nmaier** [comment]
That might be another issue
